src/libs/actions/Task.js

```diff
@@ -493,7 +493,7 @@ function setAssigneeValue(assignee, assigneeAccountID, shareDestination, isCurre
     }
 
     // This is only needed for creation of a new task and so it should only be stored locally
-    Onyx.merge(ONYXKEYS.TASK, {assignee, newAssigneeAccountID});
+    Onyx.merge(ONYXKEYS.TASK, {assignee, assigneeAccountID: newAssigneeAccountID});
```

Comment:
could I get background on this line?

Reply:
Very recent bug: https://github.com/Expensify/App/issues/20807

Reply:
https://github.com/Expensify/App/assets/9680864/294a0723-af88-44ae-8cb0-b2d5209e5caf

The bug still happens here, working on it.

Reply:
Fixed here https://github.com/Expensify/App/pull/20808/files#diff-6b302cacaeb2e1ed5d7ccfaabbb21703e6f27a4f3e932ea1412341a6e214775dL63